lib/challenge.js

```diff
@@ -19,6 +19,17 @@
   // - park
 
 // Answer Starts Here
+  'calendar'
+  Description: a calendar app that keep track of things for users
+  Every calendar has:
+    - a user
+    - a date
+    - a time
+    - an event
+  Every calendar can:
+    - add/delete new date/time/event
+    - set reminder before event date/time
+    - invite other user to certain events
```

Comment:
Completely misunderstood the question... thanks for giving me the second chance, Matt. Here are the two others I can think of.
(1) 
Description: a calendar app that help people host local events
Every calendar has:
    - host name
    - partners name
    - location
    - date/time
    - number of RSVP
Every calendar can:
    - add/delete/modify partners
    - modify location & date/time
    - change number of people RSVP

(2) 
Description: a calendar app that helps show runner to keep track of show information
Every calendar has:
    - name of the show
    - performer's name
    - location & date/time
    - maximum number of seats
    - ticket price
Every calendar can:
    - add/delete show info (including name, date, time, location, etc)
    - check available seats
    - calculate revenue
    - calculate attendance